src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -374,6 +378,10 @@ public enum RolesMappingResolution {
     public static final String TENANCY_GLOBAL_TENANT_NAME = "global";
     public static final String TENANCY_GLOBAL_TENANT_DEFAULT_NAME = "";
 
+    // Security Config Version Index feature flag
+    public static final String SECURITY_CONFIG_VERSION_INDEX_ENABLED = SECURITY_SETTINGS_PREFIX + "config_version_index.enabled";
```

Comment:
nit: more than just the index, this feature is to enable storing versions, right? can we rename the variable/setting name?